kuwala/core/canvas/src/pages/styles/data-pipeline-management.style.css

```diff
@@ -0,0 +1,18 @@
+/*!* top-left border-radius *!*/
+/*table tr:first-child th:first-child {*/
+/*    border-top-left-radius: 6px;*/
+/*}*/
+
+/*!* top-right border-radius *!*/
+/*table tr:first-child th:last-child {*/
+/*    border-top-right-radius: 6px;*/
+/*}*/
+
+/*table tr:last-child td:first-child {*/
+/*    border-bottom-left-radius: 6px;*/
+/*}*/
+
+/*!* bottom-right border-radius *!*/
+/*table tr:last-child td:last-child {*/
+/*    border-bottom-right-radius: 6px;*/
+/*}*/
```

Comment:
This file seems to be obsolete and can be deleted, no?

Reply:
This will be used later, and currently still on experiments.